proto/supply/packages/lodging_package.proto

```diff
@@ -4,9 +4,13 @@ package holms.types.supply.packages;
 option csharp_namespace = "HOLMS.Types.Supply.Packages";
 option go_package = "supply/packages";
 
+import "google/protobuf/timestamp.proto";
 import "supply/priced_offered_lodging.proto";
+import "supply/packages/lodging_package_indicator.proto";
 
 message LodgingPackage {
 	bool bookable = 1;
 	repeated holms.types.supply.PricedOfferedLodging offers = 2;
+	.holms.types.supply.packages.LodgingPackageIndicator package_id = 3;
```

Comment:
No, because it's a bit of a faux-indicator, sort of like a reservation tag.

I'll consider it but probably not likely.